index.bs

```diff
@@ -2262,21 +2262,83 @@ The WebTransport API lets the application specify multiple certificate
 hashes at once, allowing the client to accept multiple certificates for a
 period in which a new certificate is being rolled out.
 
-Unlike a similar mechanism in WebRTC, the server certificate hash API in
+Unlike a similar mechanism in [=WebRTC=], the server certificate hash API in
 WebTransport does not provide any means of authenticating the client; the fact
 that the client knows what the server certificate is or how to contact it is
 not sufficient.  The application has to establish the identity of the client
 in-band if necessary.
 
-## Pooled Sessions
+## Fingerprinting and Tracking ## {#fingerprinting}
+
+This API provides sites with the ability to generate network activity and
+closely observe the effect of this activity.  The information obtained in this
+way might be <a
+href="https://infra.spec.whatwg.org/#tracking-vector">identifying</a>.
+
+It is important to recognize that very similar networking capabilities are
+provided by other web platform APIs (such as [=fetch=] and [[webrtc]]).  The net
+adverse effect on privacy due to adding WebTransport is therefore minimal.  The
+considerations in this section applies equally to other networking capabilities.
+
+Measuring network characteristics requires that the network be used and that the
+effect of that usage be measured, both of which are enabled by this API.
+WebTransport provides sites with an ability to generate network activity toward
+a server of their choice and observe the effects.  Observations of both the
+stable properties of a network path and dynamic effect of network usage are
+possible.
+
+Information about the network is available to the server either directly through
+its own networking stack, indirectly through the rate at which data is consumed
+or transmitted by the client, or as part of the statistics provided by the API
+(see [[#web-transport-stats]]).  Consequently, restrictions on information in
+user agents is unlikely to be the only way to manage privacy risks.
```

Comment:
I'm not entirely sure what "unlikely to be the only way to manage privacy risks" means?  "unlikely to be sufficient"?  "unlikely to be effective"?

Reply:
I don't know what I was thinking when I wrote this...

```suggestion
user agents is not the only mechanism that might be needed to manage these privacy
risks.
```